ci_framework/roles/os_net_setup/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 - name: get openstack admin credentials from k8s
   vars:
-      k8s_kubeconfig: "{{ cifmw_os_net_setup_kudeconfig_location | default(cifmw_install_yamls_vars.KUBECONFIG) | default(ansible_user_dir + '/.crc/machines/crc/kubeconfig') }}"
+      k8s_kubeconfig: "{{ cifmw_os_net_setup_kubeconfig_location | default(cifmw_install_yamls_vars.KUBECONFIG) | default(ansible_user_dir + '/.crc/machines/crc/kubeconfig') }}"
```

Comment:
That's really not the way we are expecting Kubeconfig and context (even temporal token) to be used, now we have some well defined variables that are already populated with those values and are used all over the ci-framework. Try to use them please. Those variables are these ones: https://github.com/openstack-k8s-operators/ci-framework/tree/main/ci_framework/roles/openshift_login.
You can use an example here: https://github.com/openstack-k8s-operators/ci-framework/tree/main/ci_framework/roles/openshift_login

Reply:
Thanks, changed accordingly and added loading of params to 07 playbook.